ActiveLabel/ActiveLabel.swift

```diff
@@ -327,7 +327,9 @@ typealias ElementTuple = (range: NSRange, element: ActiveElement, type: ActiveTy
     // MARK: - Auto layout
 
     open override var intrinsicContentSize: CGSize {
-        return super.intrinsicContentSize
+        textContainer.size = CGSize(width: self.preferredMaxLayoutWidth, height: CGFloat.greatestFiniteMagnitude)
```

Comment:
Looks a bit weird that you override current textContainer's size. Also is this `CGFloat.greatestFiniteMagnitude` documented somehow to be used in this case to calculate appropriate of a label?